virtual-desktop/src/app/authentication-manager/login/login.component.ts

```diff
@@ -135,11 +118,65 @@ export class LoginComponent implements OnInit {
     );
   }
 
+  private handleAuthResponseMessage(authMessage: AuthResponseMessage): void {
+    if (this.isZssConnectionErrorMessage(authMessage)) {
```

Comment:
This improves the UX for ZSS but I don't like that it only works for ZSS. I'd like it even more if "ConnectionError" was a meaningful response for ANY auth handler, so that the end user would know that you couldn't login due to a server connectivity issue whether it was ZSS or another server. Seems to me that the first of any auth category that has ConnectionError should print this error, whether it is zss or not. And I'm sure that the i18n translations don't change grammatically when substituting zss for any other category name.